reposcan/repodata/repository_controller.py

```diff
@@ -293,6 +295,8 @@ def md_size(repomd, data_type):
                                          completed_repositories, total_repositories)
                         self.repo_store.store(repository)
                         repository.unload_metadata()
+                except Exception:  # pylint: disable=broad-except
+                    FAILED_IMPORT_REPO.inc()
```

Comment:
but this will silently hide the exception

nevermind, for repositories I think it will be enough to only increase counters on top level in reposcan.py

Reply:
wouldn't it be better to raise an exception right after `FAILED_IMPORT_REPO.inc()`?

Reply:
I'd not do that, it'd increment FAILED_IMPORT_REPO twice (second time on top level in reposcan.py), I think it's unnecessary